app/views/granules/download_links.html.erb

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
Is this the newly created and used file or is this one no longer necessary now that the pages are combined?

Reply:
@abbottry - this is the newly created and used file - another that is no longer necessary was deleted. ;)